tests/others_integration.txt

```diff
@@ -1,6 +1,7 @@
 sharding2
 ha
 ha_master
+ha_cases
```

Comment:
NOTE: do not forget to remove this line and add a new stage item in CI's pipeline.